docs/initialize-configuration.md

```diff
@@ -34,26 +34,50 @@ or by removing the configuration file first.
 ❌ Command Error: configuration already exists at: flow.json, if you want to reset configuration use the reset flag
 ```
 
+## Global Configuration
+
+Flow supports global configuration which is a `.flow.json` file saved in your home 
```

Comment:
Does it need to be a dotfile? I worry that users will become confused that the global version is `.flow.json` wheres the local version is `flow.json`.

Reply:
Done